engine/access/rpc/backend/backend.go

```diff
@@ -330,7 +332,7 @@ func (b *Backend) GetNodeVersionInfo(_ context.Context) (*access.NodeVersionInfo
 		Semver:               build.Version(),
 		Commit:               build.Commit(),
 		SporkId:              sporkID,
-		ProtocolVersion:      uint64(protocolVersion),
+		ProtocolVersion:      protocolSnapshot.GetProtocolStateVersion(),
```

Comment:
this will mean the value returned in the `protocol_version` field will change. I think we should deprecate the existing field ([grpc](https://github.com/onflow/flow/blob/master/protobuf/flow/entities/node_version_info.proto#L23C12-L23C28), [rest](https://github.com/onflow/flow/blob/master/openapi/access.yaml#L1232-L1234)) and add a new one that returns the `ProtocolState` version.

otherwise, anyone that's using the current version will break. it might be OK if the value was increasing, but since it's dropping from 36 to 2, I worry this will break people.

Reply:
I've deprecated the existing field, and have it set to always return 0. If you're worried about the value decreasing, maybe we should hard-code the last protocol version of 36 in there instead? Let me know what you think.